src/auth-service/utils/log-winston.js

```diff
@@ -0,0 +1,36 @@
+const winston = require("winston");
+const { combine, timestamp, printf } = winston.format;
```

Comment:
new separate util for the Winston logger used to capture and store user logs in the database. we strive to reduce the number of util files just to maintain neatness so this is just a one off due to some unit testing challenges. @BenjaminSsempala